vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/info/ProjectLinksComponent.java

```diff
@@ -33,16 +33,14 @@
  */
 @SpringComponent
 @UIScope
-public class ProjectLinksComponent extends SupportArea {
+public class ProjectLinksComponent extends PageArea {
```

Comment:
How do we know that this is in the support section?

Reply:
The component does not need to know the section in which it's employed. Additionally we currently have no difference in terms of specific styling and functionality between Support or ContentAreas so this is unnecessary